src/main/java/uk/gov/justice/hmpps/casenotes/services/CaseNoteService.java

```diff
@@ -66,39 +65,53 @@ public class CaseNoteService {
 
     public Page<CaseNote> getCaseNotes(final String offenderIdentifier, final CaseNoteFilter caseNoteFilter, final Pageable pageable) {
 
-        final List<CaseNote> sensitiveCaseNotes;
+        final List<CaseNote> dtoNotes = getCaseNotesByTypeAndSubTypes(offenderIdentifier,caseNoteFilter,pageable,caseNoteFilter.getType(), caseNoteFilter.getSubType());
+
+        final var additionalTypes = caseNoteFilter.getCaseNoteTypeSubTypes();
+        if(!StringUtils.isEmpty(additionalTypes)){
+            Arrays.stream(additionalTypes.split(",")).toList().forEach(typeLine->{
+                var types = typeLine.split("-");
+                dtoNotes.addAll(getCaseNotesByTypeAndSubTypes(offenderIdentifier,caseNoteFilter,pageable,types[0], types.length>1?types[1]:null));
```

Comment:
This will result in multiple calls to prison api - think we'll need an new endpoint to prison-api to support this

Reply:
I'd agree - bit naff to end up calling Prison API lots of times